src/index.js

```diff
@@ -50,6 +36,21 @@ function commandWrapper(pluginName, commandName) {
   };
 }
 
+function addModuleCommands(builder, module, moduleName) {
+  Object.keys(module.commands).forEach(commandName => {
+    const command = module.commands[commandName];
+
+    command.builder = command.builder || {};
+
+    builder.command(
+      command.name || commandName,
+      command.description,
+      command.builder,
+      commandWrapper(moduleName, commandName)
```

Comment:
The new version of yargs no longer hides commands with an empty description. Mup should still support that for now until version 2. Adding `builder.hide(commandName)` for commands with an empty description should work.

Reply:
I've given a look into yargs to see if there was any option as builder.hide is not working.
https://github.com/yargs/yargs/issues/851 - This fixed the issue, we can now put the description to false and the option will be hidden